app/src/androidTest/java/com/epfl/drawyourpath/login/LoginActivityTest.kt

```diff
@@ -340,4 +340,16 @@ class LoginActivityTest {
         Intents.release()
         scenario.close()
     }
+
+    @Test
+    fun getUserReturnsCurrentUser() {
+        FirebaseAuth.getUser()
+        FirebaseAuth().clearListener()
+        FirebaseAuth().onAuthStateChanged { user, error -> }
+        FirebaseAuth().onAuthStateChanged { user, error -> }
+        FirebaseAuth().loginAnonymously { _, _ -> }
+        FirebaseAuth.getUser()
+        FirebaseAuth().clearListener()
+        FirebaseAuth().signOut()
+    }
```

Comment:
I think that nothing is tested here since there is any assertion.

Reply:
These functions don't return any real data or do anything meaningful, so I don't really know what to test. The test checks that they don't crash (and most importantly improves coverage a bit).